02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,72 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Jaclyn's Webpage</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+  <div id =picture>
+    <br><br><br>
+    <h1>Hi, I'm Jaclyn Sanborn</h1>
+    <h2>Palm Springs&nbsp;&#8674;&nbsp;San Luis Obispo&nbsp;&#8674;&nbsp;<em>Austin</em></h2>
+      <hr width="45%" align="center">
+        <table>
+          <tr>
+            <td><span>Hobbies</span><br><br>
+                Playing Soccer<br>
+                Learning to play the drums</br>
+                Traveling</br></td>
+            <td><span>Favorite Comedians</span></br><br>
+                Mitch Hedberg<br>
+                Louis C.K.<br>
+                John Oliver<br></td>
+            <td><span>Favorite TV Shows</span></br><br>
+                NCIS<br>
+                Grey's Anatomy<br>
+                The Office<br></td>
+          </tr>
+        </table>
+      <br>
+      <br>
+  </div>
 
+  <div id=travel>
+    <table>
+      <tr>
+        <th colspan=3><h3>Places I'd like to visit</h3></th>
+      </tr>
+      <tr>
+        <td><img src="http://listtop10s.com/wp-content/uploads/2013/09/Cape-Town-South-Africa.jpg" alt="Cape Town, South Africa" title="South Africa"></td>
+        <td><img src="http://resources.touropia.com/gfx/d/tourist-attractions-in-morocco/chefchaouen.jpg" alt="Morocco" title="Morocco"></td>
+        <td><img src="http://traveldigg.com/wp-content/uploads/2016/06/Sheikh-Zayed-Grand-Mosque-Aerial-Photo.jpg" alt="Abu Dhabi, United Arab Emirates" title="UAE"></td>
+      </tr>
+      <tr>
+        <td>South Africa</td>
+        <td>Morocco</td>
+        <td>United Arab Emirates</td>
+      </tr>
+      <tr>
+        <td><img src="http://www.mrwallpaper.com/wallpapers/Croatia-Dubrovnik.jpg" alt ="Dubrovnik, Croatia" title="Croatia"></td>
+        <td><img src="http://famouswonders.com/wp-content/uploads/2009/03/jerusalem-old-city.jpg" alt="Jerusalem, Israel" title="Israel"></td>
+        <td><img src="http://vsem2016.org/uploaded/ha-long-bay.jpg" alt="Halong Bay, Vietnam" title="Vietnam"></td>
+      </tr>
+      <tr>
+        <td>Croatia</td>
+        <td>Israel</td>
+        <td>Vietnam</td>
+      </tr>
+    </table>
+  </div>
+
+  <div id=contact>
+    <p>
+      <br>
+        <a href="https://www.facebook.com/"><img src="http://findicons.com/files/icons/816/icons_web_2_0/128/facebook.png"></a>
```

Comment:
Great work with tables and CSS. I'd like the social media links to go directly to your page, not fb landing page. Keep it up! i'm excited to see what you build next.